vrk-kpa/suomidigi
# SUOPA-687 Groups and communities

To test: 
- `make fresh`
- To test the groups and networks views, you'll need to create few groups and few communities
   - To create groups: https://suomidigi.docker.sh/group/add
   - To create communities: https://suomidigi.docker.sh/node/add/community
- After you've got few of each, go and create a Landing page: https://suomidigi.docker.sh/node/add/landing_page
   - Give it a title, ingress and add view paragraph.
      - For the view, select Groups and for display select Groups list
      - Duplicate the paragraph from the three little dots
      - For the view, select Communities and for display select Communities list
- Save the node and check that the theme looks some what like this: https://xd.adobe.com/view/c1b40caa-b2e0-43a5-78d8-9690a43f540c-b785/screen/75431cca-7934-4a1e-a9cb-b1533b68b5ff/Suomidigi-ryhmat-koonti
